src/PointwiseFunctions/GeneralRelativity/Transform.hpp

```diff
@@ -15,11 +15,25 @@ class DataVector;
 
 /// Holds functions related to transforming between frames.
 namespace transform {
-
 //@{
-/// Transforms tensor to different frame.  Note that
-/// `Jacobian<Dest,Src>` is the same type as `InverseJacobian<Src,Dest>`
-/// and represents \f$ dx^{\rm src}/dx^{\rm dest}\f$.
+/// \ingroup GeneralRelativityGroup
+/// Transforms tensor to different frame.
+///
+/// The formula for transforming \f$T_{ij}\f$ is
+/// \f{align}
+///   T_{\bar{i}\bar{j}} &= T_{ij}
+///      \frac{\partial x^i}{\partial x^{\bar{i}}}
+///      \frac{\partial x^j}{\partial x^{\bar{j}}}
+/// \f}
+/// where \f$x^i\f$ are the source coordinates and
+/// \f$\bar{x}^i\f$ are the destination coordinates.
+///
+/// Note that `Jacobian<DestFrame,SrcFrame>` is the same type as
+/// `InverseJacobian<SrcFrame,DestFrame>` and represents
+/// \f$dx^i/dx^{\bar{j}}f$.
```

Comment:
Missing `\` before the final `f$`, and again for the other function.  I can't find the rendered documentation, but that might be because of this error.

Reply:
I pushed another fixup to fix this, but I can't find rendered documentation either.  In fact, when I go to `Files` in the documentation I can't find rendered documentation for any file in PointwiseFunctions/GeneralizedHarmonic except for Ricci.hpp and Christoffel.hpp .

Reply:
we do not require any documentation to be found through `Files`; It should be found by searching  in the search bar.  Also it is nice if you can find it starting with the appropriate namespace or doxygen group (which are found under `CodeReference/Topics`)

Reply:
I get the following when building the documentation 
```
Parsing file /var/folders/bz/3bvw0lyx5g1bvz/var/folders/bz/3bvw0lyx5g1bvzgb4cgbjp340000gn/T/spectre-doc.mBqui7r3/source/src/PointwiseFunctions/GeneralRelativity/Transform.hpp:64: warning: End of comment block while inside formula.
```

Reply:
I believe the problem rendering is that you cannot use `///` doxygen comments with multi-line latex formulae

Reply:
similar warning about the documentation at line 23 earlier in `Transform.hpp`

Reply:
Thanks Larry!  Another fixup makes my doxygen render. It's silly that Doxygen allows different comment formats, but has different functionality for different comment formats.